### PR TITLE
Fix/ct 21 ce form shared cap2

### PR DESCRIPTION
**Pull request by cliffordp**
[CT-21]

5min video of everything working (ET, ETP, CT): https://share.getcloudapp.com/geu25GkZ

[CT-21]: https://moderntribe.atlassian.net/browse/CT-21

### CODE REVIEW

File: src/Tribe/Tickets.php [hunk 1]
```diff
@@ -610,9 +610,9 @@ public function delete_ticket( $post_id, $ticket_id ) {}
 		 *
 		 * @abstract
 		 *
-		 * @param int   $post_id
-		 * @param int   $ticket
-		 * @param array $raw_data
+		 * @param int                           $post_id
```
Comment (tr1b0t) [inline src/Tribe/Tickets.php:613]: :no_entry_sign: Missing parameter comment

<sub>hash: 8c89e0fe3c7caa4fe24a2b83a8a8d153</sub>

File: src/Tribe/Tickets.php [hunk 2]
```diff
@@ -610,9 +610,9 @@ public function delete_ticket( $post_id, $ticket_id ) {}
...
+		 * @param Tribe__Tickets__Ticket_Object $ticket
```
Comment (tr1b0t) [inline src/Tribe/Tickets.php:614]: :no_entry_sign: Missing parameter comment

<sub>hash: 1bb328209a347c111e12d1315a3f9957</sub>

File: src/Tribe/Tickets.php [hunk 3]
```diff
@@ -610,9 +610,9 @@ public function delete_ticket( $post_id, $ticket_id ) {}
...
+		 * @param array                         $raw_data
```
Comment (tr1b0t) [inline src/Tribe/Tickets.php:615]: :no_entry_sign: Missing parameter comment

<sub>hash: e4d0e634b344d825e377d6e8d29e0c78</sub>

File: src/admin-views/editor/panel/list.php [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 /** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
 $tickets_handler = tribe( 'tickets.handler' );
-$total_tickets = $tickets_handler->get_total_event_capacity( $post_id );
+$total_tickets = tribe_get_event_capacity( $post_id );
```
Comment (tr1b0t) [inline src/admin-views/editor/panel/list.php:8]: :warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: a4f54cf0e79892bebcddcb77be31492a</sub>

File: readme.txt [hunk 5]
```diff
@@ -125,6 +125,7 @@ Currently, the following add-ons are available for Event Tickets:
 * Fix - Removed the `type="submit"` from the button element in the ticket quantity remove template. [ETP-224]
 * Fix - Removed the duplicate `type="button"` from the button element in the ticket quantity add template. [ETP-224]
 * Fix - Correct the `Total` value, when the WooCommerce options for currency are empty (Thousand and decimal separator, number of decimals). [ETP-231]
+* Fix - Deprecate `\Tribe__Tickets__Tickets_Handler::get_total_event_capacity()` and replace its usage with `tribe_get_event_capacity()`, which returns the correct count. [CT-21]
```
Comment (sc0ttkclark) [inline readme.txt:128]: Needs new ticket reference

Comment (cliffordp) [reply readme.txt]: done

**Pull request merged**